buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
```

Comment:
This function has no documentation attached to it. This is bad.
http://www.lispworks.com/documentation/HyperSpec/Body/m_defun.htm

Make sure to follow the documentation standards.